cc/sgx/services/fns/sgx_trusted_lib/src/trusted_worker/register_func.rs

```diff
@@ -23,16 +23,32 @@ use std::str;
 use std::vec;
 use crate::worker::{Worker, WorkerContext};
 use eigen_core::{Error, ErrorKind, Result};
-use sgx_tcrypto::{
-    rsgx_sha256_slice,
-    SgxEccHandle
-};
+
+use eigen_crypto::sign::ecdsa::{KeyPair, EcdsaKeyPair};
+use rand::Rng;
+use lazy_static::lazy_static;
+
+// TODO: For a naive implementation, now the key pair is never changed
+//       when `fns` is launched, however, in the future, the key pair
+//       may be updated by some mechanism
+lazy_static! {
+    static ref CACHED_KEY_PAIR: EcdsaKeyPair = {
```

Comment:
cache another AES key.

Reply:
OK. I've added these in the latest commitment.